authority/provisioner/acme.go

```diff
@@ -120,6 +128,8 @@ type ACME struct {
 	AttestationRoots    []byte   `json:"attestationRoots,omitempty"`
 	Claims              *Claims  `json:"claims,omitempty"`
 	Options             *Options `json:"options,omitempty"`
+	RootCRLs            []string `json:"rootCRLs,omitempty"`
```

Comment:
It would be preferable to not have to define `RootCRLs` on the provisioner. The existing properties are all meant as configuration, whereas the `RootCRLs` is currently used as state. 

If we don't expect users to have to set a different key revocation status URL, then `RootCRLs` (also see the other comment to rename it) can be unexported, and just be handled as state. Maybe all Android revocation state can be contained in a new struct as further improvement.

Reply:
I think this is based on the comment you made on discord: We may also want to be able to provide revoked keys in a way that doesn't rely on the Google endpoint for those same purposes.

I've certainly misunderstand your point so I made it as a configuration. Can be removed.